src/shady-css/tokenizer.js

```diff
@@ -25,10 +26,14 @@ class Tokenizer {
    */
   constructor(cssText) {
     this.cssText = cssText;
-    this.offset = 0;
+    this[cursorToken] = new Token(Token.type.none, 0, 0);
```

Comment:
I'm using Symbols as a de facto marker for non-public implementation.

Could you propose a documentation style example? I'm happy to go with whatever makes sense to you.